go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/btcsuite/btcutil v1.0.1
 	github.com/golang/mock v1.4.0
 	github.com/golang/protobuf v1.3.3
-	github.com/google/tink v1.3.0
+	github.com/google/tink/go v0.0.0-20200403150819-3a14bf4b3380
```

Comment:
tink moved their go.mod file to tink/go which is impacting tink imports in the project.

doing `github.com/google/tink v0.0.0` and adding
`replace github.com/google/tink => github.com/google/tink/go v0.0.0-20200403150819-3a14bf4b3380`
fails unfortunately, so the imports had to be changed.